ltr_scripts/tf_ranking_libsvm.py

```diff
@@ -81,7 +81,7 @@
 flags.DEFINE_integer("train_batch_size", 32, "The batch size for training.")
 flags.DEFINE_integer("num_train_steps", 100000, "Number of steps for training.")
 
-flags.DEFINE_float("learning_rate", 0.01, "Learning rate for optimizer.")
+flags.DEFINE_float("learning_rate", 0.1, "Learning rate for optimizer.")
```

Comment:
The learning rate determines how fast the internal weights of the model change. I picked it as it's possible to train the model faster if you use a bigger learning rate. The optimizer we use ([Adagrad](https://databricks.com/glossary/adagrad)) modifies the learning rate over time. I understood this as it could start fast and become slower/faster automatically later on, but I could be wrong here!

Reply:
Cool, so we might have to keep an eye on it, or try some alternatives at some point?

Reply:
Yes exactly. I only tried `0.1`; we could try others like `1, 0.0001`.